tensorflow/lite/toco/graph_transformations/identify_l2_normalization.cc

```diff
@@ -150,7 +136,7 @@ ::tensorflow::Status IdentifyL2Normalization::Run(Model* model,
   AddMessageF("Creating %s replacing equivalent subgraph", LogName(*l2norm_op));
 
   // Erase the subgraph that is now replaced by L2Normalization
-  model->operators.erase(FindOperator(model, square_op));
+  model->operators.erase(FindOp((*model), square_op));
```

Comment:
Just a nit: "*model" is fine to me.

Reply:
@multiverse-tf , thanks for the review i have updated as per your suggestion, kindly check.

Regards
Amit